src/collections/_documentation/common/getting_started/getting_started_template.md

```diff
@@ -0,0 +1,124 @@
+<!--
+Guideline: This page is common to all SDKs; add the name of the SDK you're documenting as well as the code samples as instructed. If you have questions, please ask Fiona or Daniel. 
+
+**The objective for this page is that a developer can quickly complete an ideal integration.**
+-->
+
+Add Sentry's <!--SDK name--> SDK to automatically report errors, exceptions, and rejections in your application. 
+
+On this page, we provide concise information to get you up and running quickly. To learn more, use the links to in-depth content.
+
+In the right place? We also offer documentation for:
+<!-- 
+Guideline: Add icons for related languages/frameworks.
+-->
+What's covered on this page:
+
+- [Install](#install)
+- [Initialize](#initialize)
+- [Verify Setup](#verify-setup)
+- [Capture Errors](#capture-errors)
+    1. [Enrich Error Data](#enrich-error-data) 
+    2. [Set the Release Version](#set-release-version)
+    
+<!--
+Guideline: Add any step to set up information that's specific to the SDK; for example, in the JavaScript example, this heading is for Source Maps. Then add the link to that heading
+-->
+
+- [Monitor Performance](#monitor-performance)
+
+## Install {#install}
+
+1.  If you already have an account, log in to Sentry. If not, establish your account:
+    1. Use your Google, GitHub, or Azure Devops account
+    2. Or, provide the basics: Your name, email, and organization, then set your password.
+2. Select <SDK> as the platform you're working with
+3. Name your project
+
+*Projects* in Sentry scope events to a distinct application in your organization, and assign responsibility and ownership. 
+
+4. Choose your installation method:
+
+<!--
```

Comment:
like all you need is 'pip install sentry-sdk' here for python (as an example).  no need for boilerplate talking about accounts/etc

Reply:
I removed the account stuff. But I think we need the install to be specified for each SDK?

This is the template, and looking at the current Getting Started, I see that each SDK is installed differently, thus the pointer to use the SDK install command. Is the current Getting Started incorrect? LMK as the goal is to have each SDK will have with a Getting Started page specific to that SDK.

A couple of examples:
`cordova plugin add sentry-cordova@0.12.3`
`$ npm install @sentry/electron@1.3.0`

Reply:
ya still need to install - i just meant this entire section can be focused on "how to add the dependency to your app"

Reply:
LMK if this version works - it's more succinct.

Reply:
looks the same to me? its still talking about accounts and sdk choices. I've already done all that if I'm on this page

Reply:
I've removed the account info, adding a sentence at the start of the page that we expect this to have already been established (and a pointer to do so).